File: aoj/volume100/csharp/A10018.cs
using System;
using System.Linq; 

class AOJ
{
	static void Main()
	{
		var chars = Console.ReadLine().ToCharArray();
		var toggledChars = chars
			.Select(c => ToToggle(c))
			.ToArray();

		Console.WriteLine(new string(toggledChars));
	}

	static char ToToggle(char ch)
	{
		if (char.IsLower(ch)) return char.ToUpper(ch);
		if (char.IsUpper(ch)) return char.ToLower(ch);
		return ch;
	}
}

File: aoj/volume100/scala/A10018.scala
object Main {
  def main(args: Array[String]) {
    val s = scala.io.StdIn.readLine
    println(s.map(ch => swapcase(ch)))
  }

  def swapcase(c:Char): Char = {
    if (c.isLower) c.toUpper
    else if (c.isUpper) c.toLower
    else c
  }
}
